lightningnetwork/lnd
# Update fee after channel becomes active, fixes #8790

## Change Description
See #8790

## Steps to Test
Have a low commitment fee, make the estimated fee high and reconnect the peer. The commitment fee should update.

## Pull Request Checklist
### Testing

### Code Style and Documentation

📝 Please see our  [Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md) for further guidance.
